src/api/entities/Asset/CorporateActions/index.ts

```diff
@@ -75,9 +75,9 @@ export class CorporateActions extends Namespace<Asset> {
   /**
    * Assign a new Corporate Actions Agent for the Asset
    *
-   * @note this may create AuthorizationRequests which have to be accepted by
-   *   the corresponding Account. An Account or Identity can
-   *   fetch its pending Authorization Requests by calling `authorizations.getReceived`
+   * @note this may create {@link AuthorizationRequest AuthorizationRequests} which have to be accepted by the `target` Identity.
```

Comment:
```suggestion
   * @note this may create {@link AuthorizationRequest | AuthorizationRequests} which have to be accepted by the `target` Identity.
```